base/docs/basedocs.jl

```diff
@@ -1312,6 +1312,27 @@ julia> Vector{Float64}(uninitialized, 3)
  6.90966e-310
  6.90966e-310
 ```
+
+    Vector{T}(nothing, m)
```

Comment:
I would prefer just splitting these into two docstrings and attach them to `Vector{T}(nothing, n)` and `Vector{T}(missing, n)` respectively. (And likewise for the `Matrix` and `Array` docstrings.)

Reply:
I wanted to do that too, but as I noted above I couldn't get it to work. Ideas?

Reply:
I meant something like:
````julia
"""
    Vector{T}(uninitialized, n)

Construct an uninitialized [`Vector{T}`](@ref) of length `n`. See [`uninitialized`](@ref).

# Examples
```julia-repl
julia> Vector{Float64}(uninitialized, 3)
3-element Array{Float64,1}:
 6.90966e-310
 6.90966e-310
 6.90966e-310
```
"""
Vector{T}(uninitialized, n)

"""
    Vector{T}(nothing, m)

Construct a [`Vector{T}`](@ref) of length `m`, initialized with
[`nothing`](@ref) entries. Element type `T` must be able to hold
these values, i.e. `Void <: T`.

# Examples
```jldoctest
julia> Vector{Union{Void, String}}(nothing, 2)
2-element Array{Union{Void, String},1}:
 nothing
 nothing
```
"""
Vector{T}(nothing, n)

"""
    Vector{T}(missing, m)

Construct a [`Vector{T}`](@ref) of length `m`, initialized with
[`missing`](@ref) entries. Element type `T` must be able to hold
these values, i.e. `Void <: T`.

# Examples
```jldoctest
julia> Vector{Union{Void, String}}(missing, 2)
2-element Array{Union{Void, String},1}:
 missing
 missing
```
"""
Vector{T}(missing, n)
````